code/modules/surgery/surgery_steps.dm

```diff
@@ -102,7 +102,8 @@ affected_limb, or location vars. Also, in that case there may be a wait between
 
 	if(ispath(tool_type)) //Tool speed modifier. This means hand & any item are 100% efficient as surgical tools.
 		tool_modifier = tools[tool_type]
-		step_duration *= tool_modifier
+		if(!(HAS_TRAIT(user, TRAIT_IMPROVISER)))
+			step_duration *= tool_modifier
```

Comment:
Just half the malus rather than completely remove it, regular tools should still be better